src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -115,4 +116,10 @@ public void parseCommand_unknownCommand_throwsParseException() {
     public void trackCommand_track() throws Exception {
         assertTrue(parser.parseCommand(TrackCommand.COMMAND_WORD + " student") instanceof TrackCommand);
     }
+
+    @Test
+    public void parseCommand_deletetag() throws Exception {
+        assertTrue(parser.parseCommand(
+                DeleteTagCommand.COMMAND_WORD + " 1" + " t/test") instanceof DeleteTagCommand);
+    }
```

Comment:
Good job adding this test to ensure the parser creates the expected instance for DeleteTagCommand!

Should this test be enhanced by checking if the instance created equals to the expected command instance for a more robust test approach? 

Perhaps instead of checking if it is an instance of DeleteTagCommand, consider changing the test to compare if the DeleteTagCommand instance created from the parser is equal to the instance expected from the user's input.
```suggestion
    @Test
    public void parseCommand_deletetag() throws Exception {
        HashSet<Tag> tagsToDelete = new HashSet<>();
        tagsToDelete.add(new Tag("test"));
        DeleteTagCommand command = (DeleteTagCommand) parser.parseCommand(DeleteTagCommand.COMMAND_WORD + " "
                + INDEX_FIRST_PERSON.getOneBased() + " " + PersonUtil.getPersonTagToDelete());
        assertEquals(new DeleteTagCommand(INDEX_FIRST_PERSON, tagsToDelete), command);
       }
```

**Note** that if this approach is taken, a method `getPersonTagToDelete()` which returns the tag prefix and tag to delete would need to be created in PersonUtil (to maintain consistency).